assets/js/components/surveys/SurveyWizardRespondentsStep.jsx

```diff
@@ -154,8 +154,8 @@ class SurveyWizardRespondentsStep extends Component {
 
     const { survey, t } = this.props
 
-    // If the survey is running, the only option is to add more respondents
-    if (survey.state == "running") {
+    // If the survey is running or paused, the only option is to add more respondents
```

Comment:
Would thinknig of an "active" state ("ogoing", or any other name) simplify things? Instead of constantly refering to "running or paused".